app/controllers/bus_stops_controller.rb

```diff
@@ -39,12 +39,12 @@ def id_search
   end
 
   def manage
-    @stops = BusStop.order(:name)
-                    .paginate(page: params[:page], per_page: 10)
+    all_stops = BusStop.order(:name)
```

Comment:
Name order seems a little arbitrary for the CSV. I might have chosen to dry it up as

```ruby
@stops = BusStop.order(:name).paginate # ..., small query
respond_to do |format|
  format.html { render :manage }
  format.csv do
    send_data BusStop.all.to_csv, filename: # ...
  end
end

Reply:
That is good idea.

Reply:
Thank